transaction/core/src/validation/validate.rs

```diff
@@ -482,897 +486,3 @@ fn check_unique<T: Eq + core::hash::Hash>(
 
     Ok(())
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Would you mind adding a comment that points at the new test files? I have often looked at the bottom of a module and thought it has no tests, only to later realize they are just somewhere else.